subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleInvocationSpec.groovy

```diff
@@ -37,19 +36,19 @@ class GradleInvocationSpec implements InvocationSpec {
     final boolean useDaemon
     final boolean useToolingApi
     final boolean expectFailure
-
-    GradleInvocationSpec(
-        GradleDistribution gradleDistribution,
-        File workingDirectory,
-        List<String> tasksToRun,
-        List<String> args,
-        List<String> jvmOpts,
-        List<String> cleanTasks,
-        boolean useDaemon,
-        boolean useToolingApi,
-        boolean expectFailure,
-        BuildAction buildAction
-    ) {
+    final File buildLog
+
+    GradleInvocationSpec(GradleDistribution gradleDistribution,
```

Comment:
Can you keep the newline after the opening braket?

Reply:
Sure.